packages/driver/src/cypress/cy.ts

```diff
@@ -122,7 +122,17 @@ const setTopOnError = function (Cypress, cy: $Cy) {
   top.__alreadySetErrorHandlers__ = true
 }
 
-export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssertions, IRetries, IJQuery, ILocation, ITimer, IChai, IXhr, IAliases, IEnsures, ISnapshots, IFocused {
+interface ICyFocused extends Omit<
+  IFocused,
+  'documentHasFocus' | 'interceptFocus' | 'interceptBlur'
+> {}
+
+interface ICySnapshots extends Omit<
+  ISnapshots,
+  'onCssModified' | 'onBeforeWindowLoad'
+> {}
```

Comment:
At first, I omit those APIs when defining `interface`s because they're not `$Cy` members. But I learned that they are used in `cy` folder files. 

So, I removed `omit` in the definitions and created new definitions here.